controllers/authpolicy_authconfig.go

```diff
@@ -65,8 +65,17 @@ func (r *AuthPolicyReconciler) desiredAuthConfig(ctx context.Context, ap *api.Au
 
 	switch obj := targetNetworkObject.(type) {
 	case *gatewayapiv1.HTTPRoute:
+		ok, err := routeGatewayHasAuthOverrides(ctx, obj, r.Client())
+		if err != nil {
+			return nil, err
+		}
+		if ok {
+			logger.V(1).Info("targeted gateway has authpolicy with atomic overrides, skipping authorino authconfig for the HTTPRoute authpolicy")
+			utils.TagObjectToDelete(authConfig)
+			r.OverriddenPolicyMap.SetOverriddenPolicy(ap)
```

Comment:
Nice. I wonder if we could do something similar (or exactly the same!) for the RLP as well.

The RLP doesn't have the same 1:1 relation to the underlying resources as the AP does with the AuthConfigs, so it would somewhere else of course. But useful for setting the status of the policy later on nevertheless.

Reply:
Yes, I think we can and probably should do something similar for RLPs also 👍